### DISCUSSION TITLE
Is there a way to use two versions of collection in same playbook ?

### DISCUSSION THREAD

**Discussion by bhati-pradeep** [ansible-community/community-topics#238] [Closed]
Hi team,
Is there a way to use version A and version B of a collection in same playbook to make use of respected features of both versions ?

**Comment by felixfontein** [comment]
I don't think there is (without using `command:` to call `ansible-playbook` recursively, which is likely not what you want to do).

**Reply by bhati-pradeep** [reply]
How to do from command line ? I have tried ansible.cfg to seperate playbooks and run with different dependencies

**Comment by briantist** [comment]
Hi @bhati-pradeep , welcome! This repository is primarily for topics related to the Ansible community that are discussed during our meetings and voted on; it's not the best place for support.

Please have a look at the Ansible community homepage which has links to discussion forums and real-time chat options:
https://www.ansible.com/community
